packages/beacon-node/src/chain/produceBlock/computeNewStateRoot.ts

```diff
@@ -35,7 +36,7 @@ export function computeNewStateRoot(
       // verifyProposer: false   | as the block signature is zero-ed
       verifyProposer: false,
       // verifySignatures: false | since the data to assemble the block is trusted
-      verifySignatures: false,
+      verifySignatures: skipRandaoVerification ?? false,
```

Comment:
if we pass this here it would also verify sync aggregate signatures

https://github.com/ChainSafe/lodestar/blob/33ca09539ba6df5ec44fb767fa59545edc89fa7e/packages/state-transition/src/block/index.ts#L65

Reply:
we can verify randao separately, it's a simple function call, I don't think passing flags down to `stateTransition` is a good idea

Reply:
this is from lighthouse https://github.com/sigp/lighthouse/pull/2740, to address the issue here https://github.com/sigp/lighthouse/issues/2612

and then they added skipping the randao verification here https://github.com/sigp/lighthouse/pull/3116